### PR TITLE
Adding issue forms and automatic labelling workflows

### PR DESCRIPTION
**Pull request by rafaspadilha**
This PR adds two issue forms (for bug report and general questions/comments) and two workflows for automatic labelling our issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      required: true
+
+  - type: dropdown
+    id: cluster-type
+    attributes:
+      label: Are you using a local or a remote (AKS) FarmVibes.AI cluster?
+      multiple: true
+      options:
+        - Local cluster
+        - Remote (ASK) cluster
```
Comment (robertomest) [inline .github/ISSUE_TEMPLATE/bug.yml:30]: ASK

File: .github/advanced-issue-labeler.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          - name: 'notebooks'
+            keys: ['Notebook execution']
+          - name: 'workflows'
+            keys: ['Workflow execution']
+      - id: [cluster-type]
+        label:
+          - name: 'local cluster'
+            keys: ['Local cluster']
+          - name: 'remote cluster'
+            keys: ['Remote (ASK) cluster']
```
Comment (robertomest) [inline .github/advanced-issue-labeler.yml:20]: AKS

**Review by robertomest — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug description
+      description: |
+        Please describe the bug you encountered in as much detail as possible.
+        If you are encountering an error, please include the full error message,
+        log file content, or the `run.reason` field if the error occurred during workflow execution.
```
Comment (renatolfc) [inline .github/ISSUE_TEMPLATE/bug.yml:42]: Should we tell users where the log files are usually located?

**Review by renatolfc — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 3 above)
Comment (rafaspadilha) [reply .github/ISSUE_TEMPLATE/bug.yml]: Added more information and the default path to the log folder to the description text.

**Pull request merged**